package.json

```diff
@@ -59,10 +59,11 @@
 		"@typescript-eslint/eslint-plugin": "^5.36.2",
 		"@typescript-eslint/parser": "^5.36.2",
 		"@vitest/coverage-c8": "^0.23.1",
-		"chokidar-cli": "^3.0.0",
 		"eslint": "^8.23.0",
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-plugin-prettier": "^4.2.1",
+		"eslint-plugin-react": "^7.31.7",
+		"eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
I assumed those were wrongfully declared as dependencies, but giving you a heads up just in case